### PR TITLE
WordAds: Prevent Crash & Hide Tab in Stats for Non-Admins

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

The "Ads" tab is currently not hidden for non-admins. This crashes the screen if clicked by a non-admin.

Screenshot 2019-07-14 at 20 31 46

#### Testing instructions

_I'd recommend to hide whitespaces when checking the code!_

On a site with WordAds, visit `/stats`, and verify that for non-admins "Ads" no longer appears.

Screenshot 2019-07-14 at 20 32 35

**Visiting the route directly**

If the user is not an admin, this should appear.

Screenshot 2019-07-16 at 17 53 44

If the user is an admin (and therefore can upgrade to WordAds), this should appear. The CTA redirects to` /earn/ads-settings/site`

Screenshot 2019-07-16 at 17 54 02

If the user has WordAds and is an admin, the page should load as normal.

Lint errors are unrelated. :)

Fixes an issue I noticed whilst working on #34597

### CODE REVIEW

**Comment by Aurorum** [comment]
@michelleweber Would you mind helping me with this copy? I don't feel it's ideal currently. :)

> "You currently do not have access to WordAds"
> "Explore WordAds!"

Screenshot 2019-07-16 at 17 54 02

cc @donpark (again, sorry!) since this is a pretty major flow flaw which breaks Calypso

**Review by donpark — APPROVED** [review]
LGTM

- All test flows worked as described.

**Comment by michelleweber** [comment]
What don't you like about the current copy? What would you like the copy to communicate that it's not currently communicating?

**Comment by Aurorum** [comment]
Hm, mainly that the user doesn't have access to WordAds...perhaps that copy isn't so unideal after all

**Comment by sixhours** [comment]
Holding off on merging this until copy is resolved.

**Comment by sixhours** [comment]
The only thing about the current copy that gives me pause is "currently", which feels a bit wordy. What about something like "WordAds are not enabled on your site"?

I like "Explore WordAds", but I don't think we usually use exclamation points or other punctuation on action buttons. I could be wrong, it just looks odd to me.

**Comment by sixhours** [comment]
All that said, copy is very easy to change after the fact, and launching this sooner rather than later would be good. :)

It's been 10 days since the last review, though; would you mind rebasing the current branch against master to make sure there haven't been other changes before I deploy this?

**Comment by Aurorum** [comment]
> would you mind rebasing the current branch against master to make sure there haven't been other changes before I deploy this?

oh dear, that was very much a complete disaster, sorry!

**Comment by michelleweber** [comment]
"WordAds are not enabled on your site" works for me. And yes, let's lose the exclamation point on the button.

**Comment by Aurorum** [comment]
Thank you, removed the exclamation mark and switched to "WordAds is _(everywhere else I could find refers to it as singular)_ not enabled on your site" in #34643 👍
